java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaLazyConfigurationException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2017, Joyent, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.joyent.manta.client;
+
+import com.joyent.manta.exception.MantaException;
+
+/**
+ * Exception for errors occurring as a result of dynamic client reconfiguration.
+ *
+ * @author <a href="https://github.com/tjcelaya">Tomas Celayac</a>
+ * @since 3.1.7
+ */
+public class MantaLazyConfigurationException extends MantaException {
```

Comment:
I'm curious why this inherits from `MantaException` and not `ConfigurationException`.